backend/mora/triggers/__init__.py

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import enum
+import logging
+import importlib
+import traceback
+import functools
+
+logger = logging.getLogger("triggers")
+
+
+def register(app):
+    for m in app.config.get("TRIGGER_MODULES", []):
+        try:
+            trigger_module = importlib.import_module(m)
+            trigger_module.register(app)
+        except Exception:
+            logger.exception("trigger code or registration error for %s", m)
+            raise
+
+
+class Trigger:
+    """ Trigger registry, retrieval, and decorator methods
+    """
+    registry = {}
+
+    @enum.unique
+    class Event(enum.Enum):
+        """ EventType for trigger registry
+        """
+        ON_BEFORE, ON_AFTER = range(2)
+
+    @classmethod
+    def run(cls, trigger_dict):
```

Comment:
The function parameters have been wrapped in a dict literal, so it's very hard to figure out what this function actually accepts. Use constants for the dict literal keys _at the very least_.